simtools/job_submission/job_manager.py

```diff
@@ -0,0 +1,207 @@
+        self.run_out_file = str(run_out_file)
+
+        self._logger.info(
+            'Submitting script {}'.format(self.run_script))
+        self._logger.info(
+            'Job output stream {}'.format(self.run_out_file+".out"))
+        self._logger.info(
+            'Job error stream {}'.format(self.run_out_file+".err"))
+        self._logger.info(
+            'Job log stream {}'.format(self.run_out_file+".job"))
+
+        if self.submitCommand.find("qsub") >= 0:
+            self._submit_gridengine()
+        elif self.submitCommand.find("condor_submit") >= 0:
+            self._submit_HTcondor()
+        elif self.submitCommand.find("seriell_script") >= 0:
+            self._submit_seriell_script()
+        elif self.submitCommand.find("more") >= 0:
+            self._submit_for_testing()
+
+    def _submit_for_testing(self):
+        """
+        Print submit script to screen for testing
+
+        """
+
+        self._logger.info('Testing script')
+
+        shellCommand = 'more ' + self.run_script
+
+        if not self.test:
+            os.system(shellCommand)
+        else:
+            self._logger.info('Testing (submit_for_testing)')
+
+    def _submit_seriell_script(self):
+        """
+        Run a job script on the command line
+        (no submission to a workload manager)
+
+        """
+
+        self._logger.info('Running script locally')
+
+        shellCommand = self.run_script + \
+            " > " + self.run_out_file + ".out"
+
+        if not self.test:
+            os.system(shellCommand)
+        else:
+            self._logger.info('Testing (seriell_script')
+
+    def _submit_HTcondor(self):
+        """
+        Submit a job described by a shell script to HTcondor
+
+        """
+
+        _condor_file = self.run_script + ".condor"
+        self._logger.info('Submitting script to HTCondor ({})'.format(
+            _condor_file))
+        try:
+            with open(_condor_file, 'w') as file:
+                file.write("Executable = {}\n".format(self.run_script))
+                file.write("Output = {}\n".format(self.run_out_file+".out"))
+                file.write("Error = {}\n".format(self.run_out_file+".err"))
+                file.write("Log = {}\n".format(self.run_out_file+".job"))
+                file.write("queue 1\n")
+        except FileNotFoundError:
+            self._logger.error(
+                "Failed creating condor submission file {}".format(
+                    _condor_file))
+
+        shellCommand = self.submitCommand + " " + _condor_file
+        if not self.test:
+            os.system(shellCommand)
+        else:
+            self._logger.info('Testing (HTcondor')
+
+    def _submit_gridengine(self):
+        """
+        Submit a job described by a shell script to gridengine
+
+        """
+
+        thisSubCmd = copy(self.submitCommand)
+        if 'log_out' in thisSubCmd:
+            thisSubCmd = thisSubCmd.replace(
+                'log_out', self.run_out_file + ".out")
```

Comment:
I can't find where we inserted ``log_out`` and ``log_err`` in the submitCommand. If we trust the user it can break easily (also I don't see why we need to trust the user). 
Either way, I think we can create the submit command directly instead of this `replace`

Reply:
This has been removed.